### PR TITLE
Resolves #410: Implement command and functionality to set the world's spawn

### LINKED ISSUE #410: No way to set the world spawn point
We can get the world spawn point, but we cannot set it. This is an important feature and will be needed when the /setworldspawn command is implemented.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Resolves issue #410

File: gomint-server/src/main/java/io/gomint/server/command/vanilla/SetWorldSpawnCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public CommandOutput execute( CommandSender sender, String alias, Map<String, Object> arguments ) {
+        CommandOutput output = new CommandOutput();
+
+        if (! ( sender instanceof PlayerCommandSender ) ) {
+            return output.fail( "Executor is required to be a player" );
+        }
+
+        EntityPlayer executor = ( EntityPlayer ) sender;
+        WorldAdapter affectedWorld = executor.getWorld();
+        Location worldSpawnLocation = ( Location ) arguments.getOrDefault( "spawnPoint", executor.getLocation() );
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/command/vanilla/SetWorldSpawnCommand.java]: Output of a BlockPositionValidator is not a Location. This line will throw a ClassCastException

**Review by geNAZt — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/command/vanilla/SetWorldSpawnCommand.java (hunk 1 above)
Comment (ghost) [reply gomint-server/src/main/java/io/gomint/server/command/vanilla/SetWorldSpawnCommand.java]: Aight. Gonna change that. Thanks.

**Comment by ghost** [comment]
This should do it.

**Pull request merged**